src/Imath/ImathFrustum.h

```diff
@@ -581,6 +657,25 @@ Frustum<T>::normalizedZToDepth (T zval) const
     }
 }
 
+template <class T>
+IMATH_CONSTEXPR14 T
+Frustum<T>::normalizedZToDepth_noexcept (T zval) const noexcept
+{
+    T Zp = zval * 2.0 - 1;
+
+    if (_orthographic)
+    {
+        return -(Zp * (_farPlane - _nearPlane) + (_farPlane + _nearPlane)) / 2;
```

Comment:
T(2) - I would make a pass over the whole file and make that consistent (it may be in the old code too!)